hil/cli/network.py

```diff
@@ -34,19 +36,74 @@ def network_delete(network):
 
 @network.command(name='show')
 @click.argument('network')
-def network_show(network):
+@click.option('--jsonout', is_flag=True)
+def network_show(network, jsonout):
     """Display information about network"""
-    q = client.network.show(network)
-    for item in q.items():
-        sys.stdout.write("%s\t  :  %s\n" % (item[0], item[1]))
+    raw_output = client.network.show(network)
+
+    if jsonout:
+        json_output = json.dumps(raw_output)
+        print(json_output)
+        return
+
+    net_table = PrettyTable()
+    net_table.title = 'NETWORK TABLE'
+    net_table.field_names = ['ATTRIBUTE', 'INFORMATION']
+
+    if 'owner' in raw_output:
+        net_table.add_row(['Owner', raw_output['owner']])
+        net_table.add_row(['', ''])
+    if 'name' in raw_output:
+        net_table.add_row(['Name', raw_output['name']])
+        net_table.add_row(['', ''])
+    if 'access' in raw_output:
+        net_table.add_row(['Access', raw_output['access'][0]])
+        net_table.add_row(['', ''])
+    if 'channels' in raw_output:
+        net_table.add_row(['Channels', raw_output['channels'][0]])
+        for i in range(1, len(raw_output['channels'])):
+            net_table.add_row(['', raw_output['channels'][i]])
+        net_table.add_row(['', ''])
+    if 'connected-nodes' in raw_output:
+        firstElement = 0
+        for node, nic in raw_output['connected-nodes'].items():
+            if firstElement == 0:
+                net_table.add_row(['Connected Nodes', node + "->" + nic[0]])
+                for i in range(1, len(nic)):
+                    net_table.add_row(['', node + "->" + nic[i]])
+                firstElement += 1
+            else:
+                net_table.add_row(['', node + "->" + nic[0]])
+                for i in range(1, len(nic)):
+                    net_table.add_row(['', node + "->" + nic[i]])
+
+    print(net_table)
 
 
 @network.command(name='list')
-def network_list():
+@click.option('--jsonout', is_flag=True)
+def network_list(jsonout):
     """List all networks"""
-    q = client.network.list()
-    for item in q.items():
-        sys.stdout.write('%s \t : %s\n' % (item[0], item[1]))
+    raw_output = client.network.list()
+
+    if jsonout:
+        json_output = json.dumps(raw_output)
+        print(json_output)
+        return
+
+    count = 0
+    net_table = PrettyTable()
+    net_table.title = 'NETWORK LIST'
+    net_table.field_names = ['Network Name', 'Network ID', 'Project Name']
+    for key1, value1 in raw_output.items():
+        for key2, value2 in value1.items():
+            if count % 2 == 0:
+                pid = value2
+            else:
+                pname = value2
+            count += 1
```

Comment:
I don't understand what's going on here. Could you use more descriptive variable names? and some comments.